elixir/apps/web/lib/web/live/settings/api_clients/components.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Web.Settings.ApiClients.Components do
+  use Web, :component_library
+
+  attr :type, :atom, required: true
+  attr :form, :any, required: true
+  attr :subject, :any, required: true
+
+  def api_client_form(assigns) do
+    ~H"""
+    <div>
+      <.input label="Name" field={@form[:name]} placeholder="Name for API client" required />
+    </div>
```

Comment:
Just a minor suggestion to add a suggestion / example here

```suggestion
    <div>
      <.input label="Name" field={@form[:name]} placeholder='E.g. "GitHub Actions"' required />
      <p class="mt-2 text-xs text-neutral-500">
        Describe what this API client is for. This can be changed later.
      </p>
    </div>
```

Reply:
👍 Good idea.  This has been updated.